### ISSUE TITLE
PaperMC exception

### ISSUE THREAD

**Issue by recon88** [Fierioziy/ParticleNativeAPI#13]
Getting an exception when using it with the latest PaperMC Server: 1.20.6-88-master@980cff9
It works fine with Spigot.
All my plugin does to the point of the exception is initializing the API (line 22 in Tools.java mentioned below).
```ParticleNativeAPI particles= ParticleNativeCore.loadAPI(plugin);```
Using v4.2.0 of your library.

```
[10:29:30 ERROR]: Error occurred while enabling AutoRegionGuard v3.0 (Is it up to date?)
java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
        at autoregionguard-3.0.0-SNAPSHOT.jar/me.yourpluginpackage.particleapi.core.ParticleNativeCore.loadAPI(ParticleNativeCore.java:41) ~[autoregionguard-3.0.0-SNAPSHOT.jar:?]
        at autoregionguard-3.0.0-SNAPSHOT.jar/com.recon.autoregionguard.util.Tools.<init>(Tools.java:22) ~[autoregionguard-3.0.0-SNAPSHOT.jar:?]
        at autoregionguard-3.0.0-SNAPSHOT.jar/com.recon.autoregionguard.AutoRegionGuard.onEnable(AutoRegionGuard.java:39) ~[autoregionguard-3.0.0-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:287) ~[paper-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[paper-1.20.6.jar:1.20.6-88-980cff9]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.20.6.jar:1.20.6-88-980cff9]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:519) ~[paper-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:604) ~[paper-1.20.6.jar:1.20.6-88-980cff9]
        at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:553) ~[paper-1.20.6.jar:1.20.6-88-980cff9]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:675) ~[paper-1.20.6.jar:1.20.6-88-980cff9]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:437) ~[paper-1.20.6.jar:1.20.6-88-980cff9]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:323) ~[paper-1.20.6.jar:1.20.6-88-980cff9]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1136) ~[paper-1.20.6.jar:1.20.6-88-980cff9]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:323) ~[paper-1.20.6.jar:1.20.6-88-980cff9]
        at java.base/java.lang.Thread.run(Thread.java:1570) ~[?:?]

 ```

**Comment by Fierioziy** [comment]
It looks like PaperMC changed the way craftbukkit classes are now packaged - without the usual `org.bukkit.craftbukkit.<version>` pattern.

I'll try to workaround this in near future, I hope it is the only change that needs to be addressed

**Comment by Fierioziy** [comment]
Fixed in version 4.3.0
